src/components/DateInput.jsx

```diff
@@ -89,7 +92,10 @@ export default class DateInput extends React.Component {
   }
 
   render() {
-    const { dateString } = this.state;
+    const {
+      dateString,
+      isTouchDevice: isTouch,
```

Comment:
I'm confused about the need for this name change. Can't we just keep this as `isTouchDevice`?